primitives/src/migrate.rs

```diff
@@ -60,3 +61,39 @@ pub fn migrate_map_rename<T: Migrate>(module: &[u8], item: &[u8], new_item: &[u8
         .filter_map(|(key, old)| Some((key, old.migrate()?)))
         .for_each(|(key, new)| put_storage_value(module, new_item, &key, new));
 }
+
+/// Migrate the keys of a double map `K1, K2` to keys of type `KN1, KN2` via `map`.
+/// The double map is located in `module::item` and the value of type `V` is ontouched.
+/// This assumes that the hashers are all the same, which is the common case.
```

Comment:
Was about to flag this but I guess you already have it covered. I guess we can add support for different hashers if/when we need it.